app/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="DrawingTheme" parent="Theme.AppCompat.DayNight.NoActionBar">
+        <item name="backgroundColor">@color/arsenic</item>
+        <item name="android:background">@color/arsenic</item>
```

Comment:
You aren't declaring your attr for sheet color in either of your themes

Reply:
technically my sheet isn't in this PR so I removed this and will include it in an appropriate PR